src/Container/PackageProxyContainer.php

```diff
@@ -66,7 +66,9 @@ private function tryContainer(): bool
 
         /** TODO: We need a better way to deal with status checking besides equality */
```

Comment:
With the new `statusIs` method, this TODO can be removed now, right?

```suggestion
```

Reply:
The problem is, that `statusIs` just checks `===`, but we also have an `assertStatus()` which has more flexibility, but throws an exception. so basically we need the same as `assertStatus()` without throwing an exception 😬

Reply:
Ah, my bad. I thought that the `statusIs` has been introduced quite recently, and before it was literally `$currentStatus === SOME_STATUS`, but that's not true. I will mark this comment as "resolved" (as in not applicable).

Reply:
We have a new `checkStatus()` method that would allow us to remove this comment, but it is currently private. I'll open a new PR to make it public.